backend/src/main/scala/com/softwaremill/adopttapir/starter/content/ContentService.scala

```diff
@@ -0,0 +1,51 @@
+package com.softwaremill.adopttapir.starter.content
+
+import com.softwaremill.adopttapir.logging.FLogging
+import com.softwaremill.adopttapir.starter.StarterDetails
+import com.softwaremill.adopttapir.template.ProjectGenerator
+
+class ContentService(projectGenerator: ProjectGenerator) extends FLogging {
+
+  def generateContentTree(starterDetails: StarterDetails): Node = {
+    val projectName = starterDetails.projectName
+    val projectAsFilesWithPaths = projectGenerator.generate(starterDetails)
+    val projectAsDirTrees = projectAsFilesWithPaths.map(gf => {
```

Comment:
shouldn't we somehow delete/cleanup the generated files?

Reply:
ah, these are just in-memory ones; but then we don't format these files

you'll probably need to extract appropriate code from `StarterService`

Reply:
Moved in-memory-files formatting to a different object, since it become not only `StarterService` responsibility.